### DISCUSSION TITLE
Fuel economy

### DISCUSSION THREAD

**Discussion by paula-fel** [transport-data/community#17] [Data needs]
For fuel economy, we desire data with these dimensions:

- service (passenger/freight)
- mode
- vehicle type
- powertrain technology

Some sources that, per scoping, provide data for this measure:

- #10 
- #16 
- #20

**Comment by cragliam** [comment]
careful these are **new vehicle** fuel economy rather than fleet. The most comprehensive analysis on fuel economy is done as part of the Global Fuel Economy Initiative (IEA, ICCT, ITF, UC Davis and UNEP) https://www.iea.org/reports/global-fuel-economy-initiative-2021

**Reply by khaeru** [reply]
@paula-fel we should probably add another discussions item in the "Sources" category for GFEI.

Structure-wise, iTEM Open Data handled the new/all vehicles distinction with an additional concept/list of codes that we called `FLEET` (open to a better name): https://github.com/transportenergy/database/blob/c92baa3944d852cbf0eaddc0b2a7982a679ac744/item/structure/base.py#L250-L264 —so data could have either:
- A _dimension_ `FLEET` containing values like `NEW`, `USED`, `_T` (total/all vehicles), possibly different for different data points; or
- An _attribute_ (attached to the entire data set) specifying that `FLEET=NEW`.

In this way, we avoid overloading the measure/concept of "fuel economy” to handle this distinction; and we can re-use `FLEET` for other measures, e.g. #6.

**Reply by khaeru** [reply]
> @paula-fel we should probably add another discussions item in the "Sources" category for GFEI.

Sorry, I see you already created it! Too slow on my part.
